enterprise/__init__.py

```diff
@@ -2,4 +2,4 @@
 Your project description goes here.
 """
 
-__version__ = "4.7.6"
+__version__ = "4.8.1"
```

Comment:
Please also add an entry in https://github.com/openedx/edx-enterprise/blob/master/CHANGELOG.rst